actions/workflows/st2_package_test.yaml

```diff
@@ -0,0 +1,83 @@
+---
+version: '2.0'
+
+st2cd.st2_package_test:
+    type: direct
+    input:
+        - hostname
+        - dns_zone
+        - instance_type
+        - environment
+        - key_name
+        - keyfile
+        - distro
+        - role
+        - release
+        - version
+        - bootstrap_branch
+        - bootstrap_script
+        - debug
+        - st2_username: test
+        - st2_password: Ch@ngeMe
+
+    task-defaults:
+        on-error:
+            - destroy_vm: <% not $.debug %>
+            - fail
+
+    tasks:
+        init:
+            action: std.noop
+            publish:
+                bootstrap_script_url: <% coalesce(
+                    $.bootstrap_script,
+                    "https://raw.githubusercontent.com/StackStorm/st2-packages/" +
+                        $.bootstrap_branch + "/scripts/st2_bootstrap.sh") %>
+            on-success:
+                - create_vm
+        create_vm:
+            action: st2cd.create_vm_role
+            input:
+                hostname: <% $.hostname %>
+                instance_type: <% $.instance_type %>
+                environment: <% $.environment %>
+                key_name: <% $.key_name %>
+                keyfile: <% $.keyfile %>
+                dns_zone: <% $.dns_zone %>
+                distro: pkg_<% $.distro %>
+                role: <% $.role %>
+            publish:
+                vm_info: <% task(create_vm).result.tasks[2].result.result[0] %>
+                vm_fqdn: <% $.hostname %>.<% $.dns_zone %>
+            on-success:
+                - get_bootstrap_script
+        get_bootstrap_script:
+            action: core.remote_sudo
+            input:
+                hosts: <% $.vm_fqdn %>
+                cmd: curl -Ss -k -o /tmp/st2_bootstrap.sh <% $.bootstrap_script_url %>
```

Comment:
any reasons to not going to a one-liner here? 
`curl -sSL https://raw.githubusercontent.com/StackStorm/st2-packages/master/scripts/st2_bootstrap.sh | sudo bash`